apps/src/studio/studio.js

```diff
@@ -1170,6 +1170,14 @@ Studio.willSpriteTouchWall = function (sprite, xPos, yPos) {
   return Studio.willCollidableTouchWall(sprite, xCenter, yCenter);
 };
 
+Studio.getWallValue = function (row, col) {
+  if (Studio.walls) {
+    return skin[Studio.walls] ? (skin[Studio.walls][row][col] << constants.WallCoordsShift): 0;
```

Comment:
perhaps i'm misreading, but don't we want the skin-specified walls to not require a shift?